R/pkg/R/column.R

```diff
@@ -235,20 +248,16 @@ setMethod("cast",
           function(x, dataType) {
             if (is.character(dataType)) {
               column(callJMethod(x@jc, "cast", dataType))
-            } else if (is.list(dataType)) {
```

Comment:
Actually this was broken code in the sense that `tojson` is not a valid function defined in base R or SparkR. I think this is a remnant from early code written in amplab-extras/SparkR-pkg. I will remove the examples above -- thats a good point